src/piyushPJun20/ArrayListMethod.java

```diff
@@ -0,0 +1,48 @@
+package piyushPJun20;
+
+import java.util.ArrayList;
+
+public class ArrayListMethod {
+	void display(ArrayList<String> name) {
+		System.out.print("Techno word index in given arrayList is ");
+		for (int index = 0; index < name.size(); index++) {
+			if (name.get(index) == "Techno") {
+				System.out.print(index + ",");
+			}
+		}
+	}
```

Comment:
Code looks good to me. You can also combine all the three methods in one. But this way also its correct because if user just wants to know the count he can directly call the count method.